app/src/main/java/com/example/sun/data/repository/source/ remote/fetchjson/ApiManager.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.sun.data.repository.source.remote.fetchjson
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import com.example.sun.data.repository.source.remote.OnResultListener
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.MalformedURLException
+import java.net.URL
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+object ApiManager {
+    private val mExecutor: ExecutorService = Executors.newFixedThreadPool(1)
```

Comment:
Sao lại để pool size = 1 vậy em? và cũng không hard code thế này nhé
```suggestion
    private val mExecutor: ExecutorService = Executors.newFixedThreadPool(1)
```